src/BotWebApi/Controllers/CandidatesController.cs

```diff
@@ -20,61 +22,119 @@ public class CandidatesController : ApiController
         [HttpGet]
         public HttpResponseMessage All()
         {
-            HttpResponseMessage response = new HttpResponseMessage(HttpStatusCode.OK)
+            return new HttpResponseMessage()
             {
-                Content = new StringContent(JsonConvert.SerializeObject(_context.Candidates)),
+                StatusCode = HttpStatusCode.OK,
+                Content = new StringContent(JsonConvert.SerializeObject(_context.Candidates, Formatting.Indented, new JsonSerializerSettings
+                {
+                    
+                })),
             };
-            response.Headers.Add("Access-Control-Allow-Origin", "*");
-            return response;
         }
 
         [HttpGet]
         public HttpResponseMessage Get(int id)
         {
             HttpResponseMessage response;
-            var foundedCandidate = _context.Candidates.First(x => x.Id == id);
-            if (foundedCandidate != null)
+            if(_context.Candidates.Any(x=>x.Id == id))
```

Comment:
don't use any
use firstordefault and check on null - so you get twice less requestes to db